backend/main/views/authentication.py

```diff
@@ -15,5 +21,38 @@ def perform_create(self, serializer):
         user.set_password(serializer.validated_data['password'])
         user.save()
 
+class CustomObtainTokenView(ObtainAuthToken):
+    def post(self, request):      
+        try:
+            user = User.objects.get(email=request.data['email'])
+            if not user.check_password(request.data['password']):
+                return Response({"error": "Incorrect password"}, status=status.HTTP_200_OK)
```

Comment:
This should not return `status=OK` since its an error, Either of these 3 is okay:  `BadRequest Unauthorized Forbidden`. Apply to all related